README.md

```diff
@@ -1,13 +1,69 @@
 # node-screenshot-machine
-
 Simple nodejs wrapper for [Screenshot Machine](http://www.screenshotmachine.com/)'s API.
 
 ## API
 
 Screenshot Machine provides a very simple url-based [API](http://www.screenshotmachine.com/apiguide.php), with config options passed as optional url params. This library provides a convenient, node-ish wrapper that makes using Screenshot Machine in your node app a snap.
 
+###Installing
```

Comment:
Make these 2nd-level, e.g. `## Installing`. As a consistency thing, keep a space between the markdown chars and the text in headings, just for easier readability in raw view.